test/testinput/bufr_bufr_backend_satwnd_amv_goes_mpi4.yaml

```diff
@@ -0,0 +1,58 @@
+time window:
+  begin: "2018-04-14T21:00:00Z"
+  end: "2023-12-15T03:00:00Z"
```

Comment:
For my own reference, why are the satellite yamls such large time windows?

Reply:
This is purposely done since we want to test several other cycles as well.  So, we gave a big range.